ci-wheel.Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG CUDA_VER=notset
 ARG LINUX_VER=notset
 
-ARG BASE_IMAGE=nvcr.io/nvidia/cuda:${CUDA_VER}-devel-${LINUX_VER}
+ARG BASE_IMAGE=nvidia/cuda:${CUDA_VER}-devel-${LINUX_VER}
```

Comment:
cc: @raydouglass to check if it's okay to drop the `nvcr.io` and pull from DockerHub? I recall there being discussion over this but I don't remember what.

Reply:
It is worth noting that conda images use DockerHub

https://github.com/rapidsai/ci-imgs/blob/7ff9ab59881f15ace5116916334a9bda65d5926f/ci-conda.Dockerfile#L9

Whereas wheels are using `nvcr.io`

Unclear whether there is a reason to use two different providers in these cases

Ideally we would try DockerHub and fallback to `nvcr.io`. Maybe there is a clean way to configure Docker to do that outside the `Dockerfile`s

Reply:
Should add switched to DockerHub because the `nvcr.io` images are not yet available whereas the DockerHub images are up

Reply:
It should be fine. If I recall correctly, we used Docker Hub originally but switched when some of the rate limits came out. But that should be all sorted now.

Reply:
Ah ok. Thanks Ray! 🙏

Yeah rate limits sounds like a plausible past issue

Also when looking in the history, noticed this PR ( https://github.com/rapidsai/ci-imgs/pull/85 ) where `nvcr.io` was used as a fallback to pickup some images deleted on DockerHub. Another issue we have dealt with before

IOW it sounds like this was done as a workaround for image availability of one form or another